public/electron.js

```diff
@@ -94,6 +125,32 @@ if (process.platform === 'darwin') {
 // Some APIs can only be used after this event occurs.
 app.on('ready', createWindow)
 
+function startSystray() {
+  tray = new Tray(path.join(__dirname, trayIcon));
+  const contextMenu = Menu.buildFromTemplate([
+    {
+      label: 'Show/Hide', 
+      type: 'normal',
+      click() {  
+        if (mainWindow.isVisible()) {
+          mainWindow.hide();
+        } else {
+          mainWindow.show();
+        }
+      }
+    },
+    { 
+      label: 'Exit', 
```

Comment:
It seems weird to only show the Exit menu when the systray is enabled. Wouldn't it be better to always show it?

Reply:
> It seems weird to only show the Exit menu when the systray is enabled. Wouldn't it be better to always show it?

This menu is shown when the user clicks on the systray.